controllers/operator_controller.go

```diff
@@ -338,3 +348,8 @@ func mapBDStatusToReadyCondition(existingBD *rukpakv1alpha1.BundleDeployment, ob
 		ObservedGeneration: observedGeneration,
 	}
 }
+
+// isBundleDepStale returns true if conditions are out of date.
+func isBundleDepStale(existingTypedBundleDeployment *rukpakv1alpha1.BundleDeployment) bool {
```

Comment:
Maybe just call it `bd` inside this function to make it a bit easier to read?

```suggestion
func isBundleDepStale(bd *rukpakv1alpha1.BundleDeployment) bool {
```